src/main/features/claim/routes/new-features-consent.ts

```diff
@@ -0,0 +1,54 @@
+import * as express from 'express'
+import { Paths as ClaimPaths } from 'claim/paths'
+import { Form } from 'forms/form'
+import { FeatureConsentResponse } from 'forms/models/featureConsentResponse'
+import { FormValidator } from 'forms/validation/formValidator'
+import { ErrorHandling } from 'shared/errorHandling'
+import { ClaimStoreClient } from 'claims/claimStoreClient'
+import { YesNoOption } from 'models/yesNoOption'
+import { User } from 'idam/user'
+import { GuardFactory } from 'response/guards/guardFactory'
+
+const claimStoreClient = new ClaimStoreClient()
+
+function renderView (form: Form<FeatureConsentResponse>, res: express.Response) {
+  res.render(ClaimPaths.newFeaturesConsent.associatedView, { form: form })
+}
+
+function checkConsentedAlready (): express.RequestHandler {
```

Comment:
Ideally we would like to use constant as there is no need to create instance every time. I have sent @singh-surjeet the following example the other day:

```
const stateGuardRequestHandler: express.RequestHandler = GuardFactory.create((res: express.Response): boolean => {
  const claim: Claim = res.locals.claim

  return claim.respondedAt !== undefined
}, (req: express.Request, res: express.Response): void => {
  res.redirect(DashboardPaths.dashboardPage.uri)
})
```

```
export default express.Router()
  .get(Paths.confirmationPage.uri,
    stateGuardRequestHandler,
    (req: express.Request, res: express.Response) => {
    ...
```